src/callbacks.jl

```diff
@@ -90,3 +90,19 @@ function callback_ub(cbdata, x::JuMP.VariableRef)
         PricingVariableUpperBound(cbdata), index(x)
     )
 end
+
+"""
+    AbstractCustomData
+
+Every custom data assigned to a pricing variable or a user cut should inherit from it.
+"""
+abstract type AbstractCustomData end
```

Comment:
Does it replace `ColunaBase.AbstractCustomData` ?

Reply:
This change was from @artalvpes commit. I forgot to ask for his review too.

Reply:
When I added it, I was not aware of ```ColunaBase.AbstractCustomData```. You can remove it.